corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -107,3 +109,6 @@ <h3 data-bind="text: $root.activeCaseType()"></h3>
     </div>
   </div>
 {% endblock %}
+{% block js %}
```

Comment:
This should be removed - js dependencies are handled in js, not html.

Reply:
:+1: the error message suggested doing this, but it also didn't work. I just left it in in case I had messed something up.